src/NuGet/Microsoft.Orleans.Client.nuspec

```diff
@@ -18,9 +18,9 @@
     <copyright>Copyright Microsoft 2015</copyright>
     <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
     <dependencies>
-      <dependency id="Microsoft.Orleans.Core" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansAzureUtils" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansProviders" version="1.0.3" />
+      <dependency id="Microsoft.Orleans.Core" version="$version$" />
```

Comment:
I'm suggesting to replace 1.0.3 with 1.0.5 in the dependency nodes because this one is a breaking change, but not put $version$ there, so that we don't automatically bump the minimum required version of dependencies when we update one or more of the packages. For example, we can do several iterations of the Templates.* packages without updating any of the packages with binaries, especially if we keep the Development package and do not directly put binaries into Templates.*.